tmt/plans/pd-e2e-plan.fmf

```diff
@@ -0,0 +1,112 @@
+# /**********************************************************************
+#  Copyright (C) 2025 Red Hat, Inc.
+#  
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#  
+#  SPDX-License-Identifier: Apache-2.0
+#  ***********************************************************************/
+
+summary: | 
+  This plan provisions a test environment for running Playwright end-to-end tests against the Podman Desktop application. 
+  It installs all required system dependencies, including GUI libraries, 
+  sets up Node.js with pnpm, and starts Podman rootless service.
+
+discover:
+    how: fmf
+
+execute:
+    how: tmt 
+
+provision: 
+  hardware: 
+    memory: ">= 16 GB"
+    cpu: 
+      cores: ">= 4"
+    disk:
+      - size: ">= 20 GB"
+
+environment:
+  TEST_PODMAN_MACHINE: true
```

Comment:
This env. var should be used only if we wanna test podman machine. Which is not case here. pls remove.